charts/openebs/network/experiments/openebs_network_delay.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Network delay between Pod and PVC belonging to a deployment/statefulset/daemonset
+kind: ChaosExperiment
+metadata:
+  labels:
+    litmuschaos.io/name: kubernetes
+  name: pod-network-delay
+  namespace: litmus
+spec:
+  definition:
+    args:
+    - -c
+    - ansible-playbook ./experiments/chaos/openebs_target_network_delay/test.yml -i /etc/ansible/hosts -i /etc/ansible/hosts
+      -vv; exit 0
+    command:
+    - /bin/bash
+    env:
+    - name: ANSIBLE_STDOUT_CALLBACK
+      value: default
+
+    - name: OPERATOR_NAMESPACE
+      value: openebs
+
+    - name: APP_NAMESPACE
+      value: default # NS of deployed app (Mandatory)
+
+    - name: APP_LABEL
+      value: '' # deployed app label (Mandatory)
+
+    - name: APP_PVC
+      value: '' # PVC name (Mandatory)
```

Comment:
If you are taking the value in the string, Put all value field in inside double quotes (" ")